js/app/dashboard/index.js

```diff
@@ -130,6 +130,21 @@ function start() {
     }
   }
 
+  // the empty tour panels are initially open
+  let expandedToursIds = []
+  tours.forEach((tour) => {if (tour.itemIds.length == 0) {expandedToursIds.push(tour['@id'])}})
+  
```

Comment:
Also a matter of style preferences. If I had to do it, I would write it like this:

```
const expandedToursIds = tours
.filter((tour) => tour.itemIds.length == 0)
.map((tour) => tour['@id'])
```